src/index.html

```diff
@@ -2,16 +2,146 @@
+          <span class="last">
+            <label>
+              I accept the term of the agreement
+              <input
+                type="checkbox"
+                name="I accept"
+                required
+              >
+            </label>
+          </span>
+        </fieldset>
+        <fieldset class="section">
+          <legend>Registration</legend>
+          <span class="rows">
+            <label for="E-mail">E-mail:
+              <input
+                type="email"
+                name="email"
+                id="E-mail"
+                placeholder="email@example.com"
+                multiple
+                required
+              >
+            </label>
+          </span>
+          <span class="last">
+            <label for="password">Password:</label>
+            <input
+              type="password"
+              id="password"
+              minlength="10"
+              maxlength="20"
+              required
+            >
+          </span>
+        </fieldset>
+        <fieldset class="section">
+          <legend>An interesting fact about you!</legend>
+          <span class="rows">
+            <label for="love_cats">Do you love cats?
+              <input
+                type="radio"
+                name="pets"
+                id="yes"
+              > <label for="yes">Yes</label>
+              <input
+                type="radio"
+                name="pets"
+                id="no"
+              > <label for="no">No</label>
+            </label>
+          </span>
+          <span class="rows"><label for="favorite_color">What is your favorite color?</label> <input type="color" id="favorite_color"></span>
+          <span class="rows"><label for="time to bed">What time do you go to bed?</label> <input type="time" id="time to bed"></span>
+          <span class="rows">
+            <label for="favorite brand of cars">What are your favorite brand of cars?</label>
+            <select
+              name="cars"
+              id="favorite brand of cars"
+              multiple
+            >
+              <option value="BMW">BMW</option>
+              <option value="Audi">Audi</option>
+              <option value="Lada">Lada</option>
+            </select>
+          </span>
+          <span class="last">
+            <label for="rate our work">How do you rate our work?</label>
+            <input
+              id="rate our work"
+              value="0"
+              type="range"
+              min="0"
+              max="10"
+            >
+          </span>
+        </fieldset>
+        <fieldset class="section">
+          <legend>Additional info:</legend>
+          <span class="rows">
+            <label for="Comments">
+              Comments:
+              <textarea
+                id="Comments"
+                name="Comments"
+                cols="20"
+                rows="2"
+              >
+              </textarea>
```

Comment:
![image](https://user-images.githubusercontent.com/98764641/224285283-d2d6a0f7-afa2-4d70-a077-5c4fcc30f1c8.png)

Reply:
please look at the checklist before request a review